main.go

```diff
@@ -196,6 +199,21 @@ func loadConfig(configFile string) (Config, error) {
 	return loadConfigContent(configData)
 }
 
+func loadEnvVars(c *Config) {
+	if instanceName, ok := os.LookupEnv("SERVICENOW_INSTANCE_NAME"); ok {
+		(*c).ServiceNow.InstanceName = instanceName
+	}
+	if userName, ok := os.LookupEnv("SERVICENOW_USERNAME"); ok {
+		(*c).ServiceNow.UserName = userName
+	}
+	if password, ok := os.LookupEnv("SERVICENOW_PASSWORD"); ok {
+		(*c).ServiceNow.Password = password
+	}
+	if incidentField, ok := os.LookupEnv("SERVICENOW_INCIDENT_FIELD"); ok {
```

Comment:
I'd stick with the same name here, even if it's longer:
```suggestion
	if incidentField, ok := os.LookupEnv("SERVICENOW_INCIDENT_GROUP_KEY_FIELD"); ok {
```